indigo/indigo/src/main/scala/indigo/shared/datatypes/RGBA.scala

```diff
@@ -68,6 +68,25 @@ object RGBA:
   val None: RGBA   = White
   val Zero: RGBA   = RGBA(0, 0, 0, 0)
 
+  // https://en.wikipedia.org/wiki/X11_color_names
+  val Coral: RGBA     = fromHexString("#FF7F50")
```

Comment:
I figured using hex strings would be cleaner but I can rewrite this section to use doubles if needed.

Reply:
Hex is fine thanks.